src/exercise4/exercise2.js

```diff
@@ -0,0 +1,12 @@
+// Write a JavaScript function to check if a given number is prime or not.
+
+function isPrime(number) {
+    if(number === 2) return true;
+    if(number % 2 === 0) return false;
+
+    for(let i = 3; i <= Math.sqrt(number); i += 2) {
+        if(number % i === 0) return false;
```

Comment:
To reduce the lines of codes , Here you may use ternary operator instead of if .